zipkin-api.yaml

```diff
@@ -139,12 +140,24 @@ paths:
         required: true
         description: the 64-bit hex-encoded id of the trace as a path parameter.
         type: string
+      - name: raw
```

Comment:
`required` is required. 

I guess it would be a good idea to print validation errors in `npm test`.